alerts.go

```diff
@@ -231,56 +238,168 @@ func formatCheckMessage(msg string) string {
 
 func doAlertsRetrieve(c *cli.Context) error {
 	client := newMackerelFromContext(c)
+	isClosed := c.Bool("withClosed")
```

Comment:
s/isClosed/withClosed/

Reply:
- this doesn't work because flag definition changes to `with-closed`.
- variable name is still `isClosed` !